dist/index.js

```diff
@@ -8922,20 +8926,17 @@ const h2 = (text) => `## ${text}`;
 const h3 = (text) => `### ${text}`;
 const link = (text, href) => `[${text}](${href})`;
 const code = (text) => `\`${text}\``;
+const strikeCode = (text) => `\~\`${text}\`\~`;
```

Comment:
I like this way of representing excluded labels 😆
To match the other helpers, you could decompose into

```suggestion
const strike = (text) => `\~${text}\~`;
```

and then call it with

```js
.map(x => strike(code(x)))
```